pyaedt/modules/PostProcessor.py

```diff
@@ -945,21 +945,23 @@ def available_display_types(self, report_category=None):
 
     @pyaedt_function_handler()
     def available_quantities_categories(
-        self, report_category=None, display_type=None, solution=None, context="", is_siwave_dc=False
+        self, report_category=None, display_type=None, solution=None, context=None, is_siwave_dc=False
     ):
         """Compute the list of all available report categories.
 
         Parameters
         ----------
         report_category : str, optional
-            Report Category. Default is `None` which will take first default category.
+            Report Category. Default is ``None`` which will take first default category.
```

Comment:
```suggestion
            Report category. The default is ``None``, in which case the first default category is used.
```
Is "first" necessary here?

Reply:
@PipKat honestly I don't know because I didn't touch this line :)